site/docs/concepts/config-policies.md

```diff
@@ -188,3 +188,29 @@ cd fybrik
 helm install fybrik-crd charts/fybrik-crd -n fybrik-system --wait
 helm install fybrik charts/fybrik --set global.tag=master --set global.imagePullPolicy=Always -n fybrik-system --wait
 ```
+
+### How to add expiry date to policies
```

Comment:
How to add start and/or expiry dates to policies

Reply:
Changed